README.md

```diff
@@ -21,7 +21,7 @@ ORAS has several sub-projects, as outlined in [ORAS.land][oras-land]
 
 ## Roadmap
 
-The [RoadMap](./ROADMAP.md) lists the planned features and timeline. Everyone is welcome to raise a proposal or submit a feature request.
+The [RoadMap](./Roadmap.md) lists the planned features and timeline. Everyone is welcome to raise a proposal or submit a feature request.
```

Comment:
It's a nit, and it doesn't affect the casing of the link, but RoadMap is typically cased Roadmap. 
@Mr-DG-Wick, if you could quickly update the casing of [RoadMap] to [roadmap], it would reduce more process for an issue and PR.

Reply:
Thanks! @SteveLasker